### DISCUSSION TITLE
SD card access

### DISCUSSION THREAD

**Discussion by GustavoSchip** [rust3ds/ctru-rs#191] [Q&A] [answered]
I haven't found a way to access the SD card using ctru-rs, could someone help me?
(No search results in the issues and discussions.)

Thanks 🤗

**Comment by Meziu** [comment, marked as answer]
> I haven't found a way to access the SD card using ctru-rs, could someone help me?

The SD card is automatically accessible with normal `std::fs` commands, since the connection is established by default via `libctru`. For example, to access the `/3ds` folder you only need to do:

```rust
std::fs::read_dir("/3ds");
```

To access a custom RomFS bundled with the executable, have a look at the `romfs` example.

I hope this helps 👍

Edit: although slightly more complicated than other examples, this functionality is shown in the `file-explorer` example.

**Reply by GustavoSchip** [reply]
Thanks for the insight!

I indeed found that romfs:/ worked! And thanks for explaining that is already easily possible! 👍 

I'll try it out right now 🤗
